CHANGELOG.md

```diff
@@ -1,5 +1,42 @@
 # Changelog
 
+## [0.4.7] - 2025-06-17
+[PyPI](https://pypi.org/project/ucc/0.4.7/) | [GitHub](https://github.com/unitaryfoundation/ucc/releases/tag/v0.4.7)
+
+In v0.4.7, we are celebrating a triumphant [UnitaryHACK](https://unitaryhack.dev/) 2025, with exciting new contributions from lovely new contributors -- and a whole slew of updates and improvements by the maintainers to support their work!
+
+### ✨ Key Features
+First external compiler pass port by new contributor @ACE07-Sev 🎯: Integrated a gate-reducing pass from [BQSKit](https://github.com/BQSKit/bqskit) (#421).  
```

Comment:
Are the authors flipped here? I believe @ACE07-Sev authored the MPS encoding pass and @WolfLink authored the BQSKit pass.